docs/gambit/mutation-verifier.md

```diff
@@ -0,0 +1,99 @@
+
+The tool expects a configuration file (extension `.conf` is required)
+which is a JSON object.
+Here is an example configuration file:
+
+```
+{
+    "project_folder" : "Test/10Power",
+    "run_script" : "Test/10Power/runDefault.sh",
+    "gambit" : {
+        "filename" : "Test/10Power/TenPower.sol",
+        "solc" : "solc8.10"
+    },
+    "staging" : "master",
+    "manual_mutations" : {
+        "path/to/original/file.sol" : "path/to/manual/mutations/",
+        ...
+    },
+    ...
+}
+```
+
+Required Keys for the JSON Configuration File:
+- `"project_directory"` : the directory containing the original CVT project on which to perform mutation testing
+- `"run_script"` : the bash script used to run verification on the original project, usually `project_directory/run.sh` or similar
+- `"gambit"` : the JSON configuration element for invoking Gambit. May be a path to a gambit configuration file
+or the explicit JSON element contained therein. Refer to the open source Gambit repository for more details regarding Gambit configuration files.
+
+Optional Keys for the JSON Configuration File:
+- `"num_threads"` : the maximum number of threads to use for verification, as an integer
+- `"manual_mutations"` : optionally supplement the random mutant generation with your own manually-written mutants.
+Expects a JSON object whose keys are the paths to the original files and whose values are paths to directories containing
+manually-written mutants as `.sol` files. **IMPORTANT:** any manual mutations files provided must follow the naming
+convention `OriginalFileName.<unique-name>.sol`, where `unique-name` is a string ID unique with respect to the other
+manual mutants. It is recommended to use `mN` for brevity, where `N` is a unique integer.
+- `"offline"` : run mutation testing without internet connection, skipping the UI output and other web functions.
+Expects a boolean and defaults to `false`.
+
+Additional Optional Flags for Debugging
+- `"staging"` : if your run script does not already have `--staging`, you can also add it to Gambit.
+  Similar to CVT, you can provide the
+  branch name for running mutant verification on `--staging`.
+We support `"staging" : true` as an alternative to `"staging" : "master"`.
+Omitting this key will cause verification to run locally
+  (unless the run script has it).
+- `"use_cli_certora_run"` : Use CLI `certoraRun` rather than `certoraRun.py`. Expects a boolean and defaults to `false`.
+
+For implementation details regarding the generation of mutants,
+  refer to the open source repository
+  for [Gambit](https://github.com/Certora/gambit).
+
+## Visualization
+
+The mutation verification results are
+  summarized in an user-friendly visualization.
+[Here](https://mutation-testing-beta.certora.com/reports/mutation?id=c7c659d7-d500-46f2-acf1-1392eee714b5&anonymousKey=f4b40ba6-2160-4993-9f50-02625b291cae) is an example summary
+  for the [Borda example](https://demo.certora.com/?Borda).
```

Comment:
This takes some time to load for me.  It's probably worth including a screenshot here.

Reply:
very strange.